product_multi_company/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module sets the default company taxes for all the existing companies when
+a product is created. It also adds a button to set all the taxes from other
+companies matching them by tax code.
```

Comment:
This description doesnt seem to be the right one. Don't you think ?

Reply:
Indeed, it is very strange, I don't understand where this came from...
I fixed this anyway.